static/skin/index.js

```diff
@@ -265,16 +271,20 @@ const kiwixServe = (function() {
         await fetch(query).then(async (resp) => {
             const data = new window.DOMParser().parseFromString(await resp.text(), 'application/xml');
             let optionStr = '';
-            data.querySelectorAll('entry').forEach(entry => {
-                const title = getInnerHtml(entry, 'title');
-                const value = getInnerHtml(entry, valueEntryNode);
-                const hfTitle = humanFriendlyTitle(title);
-                if (valueEntryNode == 'language') {
-                    languages[value] = hfTitle;
-                }
-                optionStr += (hfTitle != '') ? `<option value="${value}">${hfTitle}</option>` : '';
-            });
-            document.querySelector(nodeQuery).innerHTML += optionStr;
+            const entryList = data.querySelectorAll('entry');
+            const nodeQueryElem = document.querySelector(nodeQuery);
+            if (entryList && nodeQueryElem) {
```

Comment:
When `nodeQueryElem == null` doesn't the `loadAndDisplayOptions()` function become a no-op? If so why run `fetch(query)` at all?